src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Bignum.cs

```diff
@@ -3,33 +3,23 @@
 
 using System;
 using System.Runtime.InteropServices;
-
 using Microsoft.Win32.SafeHandles;
 
 internal static partial class Interop
 {
-    internal static partial class libcrypto
+    internal static partial class Crypto
     {
-        [DllImport(Libraries.LibCrypto)]
-        internal static extern void BN_clear_free(IntPtr a);
-
-        [DllImport(Libraries.LibCrypto)]
-        private static extern IntPtr BN_bin2bn(byte[] s, int len, IntPtr zero);
+        [DllImport(Libraries.CryptoNative)]
+        internal static extern void BigNumDestroy(IntPtr a);
 
-        [DllImport(Libraries.LibCrypto)]
-        private static extern unsafe int BN_bn2bin(SafeBignumHandle a, byte* to);
+        [DllImport(Libraries.CryptoNative)]
+        private static extern IntPtr BigNumFromBinary(byte[] s, int len);
```

Comment:
Yeah, I think I wrote that before System.Security.Cryptography.Native existed.

And I was probably being lazy with "oh, the lifetime threat is small" vs calling SetHandleAsInvalid.  Not things I'd let others get away with.

::smacks self on the hand:: Bad Jeremy.